src/js/read.js

```diff
@@ -8,10 +8,27 @@ if (!readJSConfig.cb){ readJSConfig.cb = function(){
     Read.js assumes:
     1) The article width never exceeds the viewport width
 */
+
+function get_text(element) {
```

Comment:
Please add the get_text function as a public method to the readJS object.
This would prevent any global namespace conflicts with any other JS libraries/code that is on the page.